src/server/editorServices.ts

```diff
@@ -444,10 +444,10 @@ namespace ts.server {
             if (!this.eventHandler) {
                 return;
             }
-            this.eventHandler(<ProjectLanguageServiceStateEvent>{

```

Comment:
Changing the type of `ProjectServiceEventHandler` would require changing some other things; I want to avoid that in this PR. How about just a local variable?